docs/src/components/B2BIdentityJourney.css

```
@@ -0,0 +1,248 @@
+.uc-b2b-roadmap {
+  --uc-node-w: 8.7rem;
+  --uc-node-h: 5rem;
+  --uc-icon-size: 4.35rem;
+  --uc-gap-x: 0.9rem;
+  --uc-gap-y: 1rem;
+  margin: 1.1rem 0 1.45rem;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)